ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -176,7 +178,14 @@ export default class UnlockPage extends Component {
               </div>
             ) : null}
           </div>
-          <h1 className="unlock-page__title">{t('welcomeBack')}</h1>
+          <Text
+            data-testid="unlock-page-title"
```

Comment:
Can we add the polymorphic as prop here so it still renders an h1 element?
```suggestion
            data-testid="unlock-page-title"
            as="h1"
```